docs/communication.md

```diff
@@ -3,12 +3,12 @@
   "node": {
     "label": "Custom messages",
```

Comment:
The custom messages/Communication page is **both** part of Core and Client section. But here I see it under the Luigi Client section/category. Maybe it would be more suitable to put it under Navigation category, or some other place you see fits better?